### PR TITLE
Ensure nil is not written to response-chan while retrieving state …

### PR DESCRIPTION
**Pull request by shamsimam**
…for invalid service-ids

### CODE REVIEW

File: waiter/src/waiter/scheduler.clj [hunk 1]
```diff
@@ -513,7 +513,7 @@
                          (if service-id
                            (let [scheduler-state (service-id->state scheduler service-id)
                                  health-check-context (get current-state service-id)]
-                             (async/>! response-chan (merge scheduler-state health-check-context)))
+                             (async/>! response-chan (merge {} scheduler-state health-check-context)))
```
Comment (sradack) [inline waiter/src/waiter/scheduler.clj]: Would we be safer with an integration test that ensured Waiter didn't crash if we passed an invalid service id?

Comment (shamsimam) [reply waiter/src/waiter/scheduler.clj]: Will add.

Comment (shamsimam) [reply waiter/src/waiter/scheduler.clj]: Done.

**Comment by shamsimam** [comment]
RTM

**Pull request merged**